invenio_jsonschemas/ext.py

```diff
@@ -165,6 +167,17 @@ def path_to_url(self, path):
         ).build(
             'schema', values={'path': path}, force_external=True)
 
+    @cached_property
+    def loader(self):
+        """Loader used by `JsonRef.replace_refs`."""
+        try:
+            loader_ = import_string(self.app.config['JSONSCHEMAS_LOADER_CLS'])
+        except ImportError:
+            loader_ = json_loader_factory(jsonresolver.JSONResolver(
```

Comment:
The problem I see with the default loader is that when we try to resolve a schema like:

```
{'$ref': 'http://localhost:5000/schemas/records/hep.json'}
```

It gets stuck [here](https://github.com/gazpachoking/jsonref/blob/3fe0e4d386c8c172f945f7f65b2a937ad9c59d56/jsonref.py#L298) and I think it might be because it keeps calling the same URL endpoint again and again (even though I cannot see signs of an infinite loop) - the process just gets stuck locally.

Instead, when we use our own, the `url_map` gets altered here https://github.com/inveniosoftware/invenio-jsonschemas/blob/master/invenio_jsonschemas/jsonresolver.py#L40 so that that `get_schema` function gets called.